projects/plugins/jetpack/modules/search/instant-search/components/search-result-multisite.jsx

```diff
@@ -0,0 +1,109 @@
+/** @jsx h */
```

Comment:
I noticed that this component shares a lot of code with the Expanded result type. Would it make more sense to add this functionality to the Expanded result type format instead of introducing a new one?

(Note: I haven't been able to test this out on a test environment, so please take this comment with a grain of salt.)

Reply:
I was not sure how to balance this if it will be too many conditionals inside the Expanded component, I will give it a try.

Reply:
I updated the expanded version to account for multi site search as well.